packages/back-end/src/util/handler.ts

```diff
@@ -121,3 +125,36 @@ export function getBuild() {
 
   return build;
 }
+
+export function applyPagination<T>(
```

Comment:
We'll need to handle this better for Cloud. Either use a non-paginated endpoint, or build pagination into the proxy. If using pagination, what happens if the query results change between pagination requests?

Reply:
I think for Cloud, we'll just query Mongo directly instead of going through the API